src/i18n.js

```diff
@@ -0,0 +1,163 @@
+import i18n from "i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18n.use(LanguageDetector).init({
+    resources: {
+        kn: {
```

Comment:
it would be good to split kn and en to separate files, sub folder and include here

Reply:
Okay will move to a diif file